### PR TITLE
[Merged by Bors] - chore: tidy various files

### PR DESCRIPTION
**Pull request by Ruben-VandeVelde**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Algebra/Star/StarRingHom.lean [hunk 1]
```diff
@@ -160,10 +160,7 @@ end
 /-- The composition of non-unital ⋆-ring homomorphisms, as a non-unital ⋆-ring homomorphism. -/
 def comp (f : B →⋆ₙ+* C) (g : A →⋆ₙ+* B) : A →⋆ₙ+* C :=
   { f.toNonUnitalRingHom.comp g.toNonUnitalRingHom with
-    map_star' := fun a => (calc
-      (f ∘ g) (star a) = f ( g (star a)) := rfl
-      _ = star (f (g a)) := by rw [map_star, map_star]
-      _ = star ((f ∘ g) a) := rfl )}
+    map_star' := fun a => by simp [Function.comp_def, map_star, map_star] }
```
Comment (grunweg) [inline Mathlib/Algebra/Star/StarRingHom.lean:163]: Can you write `map_star' a := by simp [Function.comp_def, map_star, map_star]` here?

**Review by grunweg — COMMENTED** [review]
Some brief afternoon procrastination, but at least this means a PR got reviewed :-) Thanks for doing this, all looks very nice! Feel free to maintainer merge on my behalf with my small comment answered.

File: Mathlib/Algebra/Star/StarRingHom.lean (hunk 1 above)
Comment (Ruben-VandeVelde) [reply Mathlib/Algebra/Star/StarRingHom.lean:163]: No, that's only for the `where` syntax, not the `:= {}` syntaxt

**Comment by Ruben-VandeVelde** [comment]
maintainer merge

File: Mathlib/Algebra/Star/StarRingHom.lean (hunk 1 above)
Comment (grunweg) [reply Mathlib/Algebra/Star/StarRingHom.lean:163]: :bulb: Thanks, TIL!

**Comment by riccardobrasca** [comment]
Thanks!

bors merge

**Pull request closed without merging**